libs/ui/elements/src/lib/apis-list/apis-list.component.spec.ts

```diff
@@ -1,8 +1,9 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing'
 
 import { ApisListComponent } from './apis-list.component'
-import { ApisListItemComponent } from '../apis-list-item/apis-list-item.component'
```

Comment:
remove the api-list-component

Reply:
Ok, done. Then would you loop over the cards in `DataApisComponent` or even get rid of this component and use them directly in `RecordMetadataComponent`?

Reply:
I think `DataApisComponent` was there because of the tabs. I'm not sure but maybe we could do it directly in `RecordMetadataComponent`.
`DataApisComponent` or `ApiListComponent` don't bring much value

Reply:
For sure, they don't add much logic. For now, I'm keeping `DataApisComponent` in the UI PR (#196), so that `RecordMetadataComponent` does not become too massive. It's rather a presentational component than a smart component though. Don't hesitate to get back to this topic in the other PR to discuss what makes most sense.